address-space-controller/src/main/java/io/enmasse/controller/common/KubernetesHelper.java

```diff
@@ -386,7 +387,9 @@ private void createRoleBinding(String name, String namespace, String refKind, St
 
         for (Subject subjectEntry : subjectList) {
             JsonObject subject = new JsonObject();
-            subject.put("apiGroup", "rbac.authorization.k8s.io");
+            if (isOpenShift) {
+                subject.put("apiGroup", "rbac.authorization.k8s.io");
```

Comment:
Is this not the correct api group for kubernetes as well?

Reply:
@lulf setting the apigroup here caused this exceptions:
````
2018-06-18 11:46:17.405 WARN  KubernetesHelper:345 - Error performing POST on /apis/rbac.authorization.k8s.io/v1beta1/namespaces/bosch/rolebindings: 422, 
{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":
"RoleBinding.rbac.authorization.k8s.io \"addressspace-admins\" is invalid: 
subjects[0].apiGroup: 
Unsupported value: \"rbac.authorization.k8s.io\": 
supported values: \"\"","reason":"Invalid","details":{"name":"addressspace-admins","group":"rbac.authorization.k8s.io","kind":"RoleBinding","causes":[{"reason":"FieldValueNotSupported","message":"
Unsupported value: \"rbac.authorization.k8s.io\": supported values: \"\"","field":"subjects[0].apiGroup"}]},"code":422}
````

Reply:
@calohmn Ok, interesting. We can remove this once OpenShift 3.7 support goes away.